src/components/JobDetailPage.tsx

```diff
@@ -160,22 +237,39 @@ const JobDetailPage: FunctionComponent<IProps> = props => {
         mx='auto'
         my={2}
         borderRadius='3px'
+        position='relative'
       >
-        {SQL ? (
-          SQL.split('\n').map((line, i) => {
-            const extraClass = line.trim().startsWith('--') ? _SQLComment : ''
-
-            return (
-              <StyledTypographySQL key={i} className={extraClass}>
-                {line}
-              </StyledTypographySQL>
-            )
-          })
-        ) : (
-          <StyledTypographySQL align='center'>
-            There is no SQL for this job at this time.
-          </StyledTypographySQL>
+        {!!SQL && SQL !== '' && (
+          <StyledExpandButton color='secondary' onClick={() => setSQLModalOpen(true)} />
+        )}
+        {!!SQL && SQL !== '' && (
+          <Modal aria-labelledby='modal-title' open={SQLModalOpen}>
+            <div className={SQLModalContainer}>
+              {/* Need this extra container for the absolutely-positioned elements */}
+              <StyledCloseIcon fontSize='large' onClick={() => setSQLModalOpen(false)} />(
+              <Typography
+                color='secondary'
+                className={copyToClipboard}
+                onClick={() => {
+                  if (SQL) {
+                    navigator.clipboard.writeText(SQL)
```

Comment:
Works well. I think it'll be useful for users to have some acknowledgement that the content has been copied, though, an alert that fades away or something. I'll put a ticket in the backlog

Reply:
Yes I agree. Thanks for adding the ticket! @jhubley